nestkernel/mask_impl.h

```diff
@@ -105,7 +105,7 @@ BoxMask< D >::outside( const Box< D >& b ) const
   // so we don't know if it is an actual problem.
   for ( int i = 0; i < D; ++i )
   {
-    if ( ( b.upper_right[ i ] < min_values_[ i ] ) || ( b.lower_left[ i ] > max_values_[ i ] ) )
+    if ( ( b.upper_right[ i ] < min_values_[ i ] ) or ( b.lower_left[ i ] > max_values_[ i ] ) )
```

Comment:
Wouldn't this version with less parentheses be equivalent? The same comments applies to other conditions within this file. 

```suggestion
    if ( b.upper_right[ i ] < min_values_[ i ] or  b.lower_left[ i ] > max_values_[ i ] )
```